src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
-body {
+$main-text-color: #060b35;
+$secondary-color: #616070;
+$button-color: #00acdc;
+$white-color: #fff;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)